### PR TITLE
Nazarov_Anton/lesson-1

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/test/java/lambda/part1/exercise/Lambdas01Exercise.java [hunk 1]
```diff
@@ -21,7 +24,12 @@ public void sortPersonsByAge() {
                 new Person("name 2", "lastName 1", 30)
         };
 
-        // TODO use Arrays.sort
+        Arrays.sort(persons, new Comparator<Person>() {
+            @Override
+            public int compare(Person o1, Person o2) {
+                return o2.getFirstName().compareTo(o1.getFirstName());
```
Comment (senia-psm) [inline src/test/java/lambda/part1/exercise/Lambdas01Exercise.java:30]: See method name. `sortPersonsByAge`.

File: src/test/java/lambda/part1/exercise/Lambdas02Exercise.java [hunk 2]
```diff
@@ -18,7 +22,7 @@ public void sortPersonsByAge() {
                 new Person("name 2", "lastName 1", 30)
         };
 
-        // TODO use Arrays.sort
+        Arrays.sort(persons, (o1, o2) -> Integer.valueOf(o1.getAge()).compareTo(o2.getAge()));
```
Comment (senia-psm) [inline src/test/java/lambda/part1/exercise/Lambdas02Exercise.java:25]: Use `Comparator.comparingInt` to avoid boxing.

**Comment by senia-psm** [comment]
Don't close `PR`s.

**Comment by ghost** [comment]
@senia-psm It's a mistake (my fault with git). This PR is similar with upper #162 .

**Comment by senia-psm** [comment]
Use `PR` number to refer another `PR` like this: #162

File: src/test/java/lambda/part1/exercise/Lambdas02Exercise.java (hunk 2 above)
Comment (MortyMerr) [inline src/test/java/lambda/part1/exercise/Lambdas02Exercise.java]: May i ask a question here?

**Pull request closed without merging**